docs/pages/management/admin/labels.mdx

```diff
@@ -1,405 +1,323 @@
 ---
-title: Labels
+title: Add Labels to Resources
 description: Labeling Teleport resources
 ---
 
 Teleport allows you to add arbitrary key-value pairs to applications, servers,
-databases, and other resources in your cluster. You can use these to filter
-resources when making queries via `tctl` and `tsh` as well as to limit the
-resources that Teleport roles can access.
+databases, and other resources in your cluster. You can use labels to do 
+the following:
 
-This guide explains how to add labels to Teleport resources.
+- Filter the resources returned when running `tctl` and `tsh` commands.
+- Limit the resources that Teleport roles can access.
 
-There are two kinds of labels:
+This guide demonstrates how to add labels to enrolled server resources.
+However, you can use these instructions to add labels for any type 
+of resource managed by Teleport.
 
-### Static labels
+## Static and dynamic labels
 
-Static labels are hardcoded in a `teleport` daemon's configuration file and do
-not change over time while the `teleport` process is running. An example of a static
-label is a Node's environment, e.g., `staging` vs. `production`.
+The labels you assign to resources can be **static labels** or **dynamic labels**.
 
-### Dynamic labels
+- Static labels are hardcoded in the `teleport` configuration file and don't change 
+while the `teleport` process is running. For example, you might use a static label to identify 
+the resources in a `staging` or `production` environment.
+- Dynamic labels—also known as **commands-based labels**—allow you to generate labels at runtime. 
+With a dynamic label, the `teleport` process executes an external command on its host at 
+a configurable frequency and the output of the command becomes the label value.
 
-Dynamic labels, also known as **label commands**, allow you to generate labels
-at runtime. The `teleport` daemon will execute an external command on its host
-at a configurable frequency and the output of the command becomes the label's
-value.
+You can add multiple static and dynamic labels for the same resource.
+However, if you add static labels that use the same key but with different values, only the first 
+key-value pair is used.
```

Comment:
This is kind of surprising to me. Have we validated that this is true?

Reply:
In my test environment, the first key-value was used. Additional, keys with the same name were ignored. If overwritten, I would have expected the last entry to be used.

Reply:
I'm not sure how you were even able to _add static labels that use the same key but with different values_.

If I try, Teleport refuses to start:

```
$ teleport start -c ~/t/teleport.yaml
ERROR: failed parsing the config file: yaml: unmarshal errors:  line 37: key "foo" already set in map
```

```
 33 ssh_service:
 34   enabled: "yes"
 35   labels:
 36     foo: bar
 37     foo: baz
```